utils/FIS/FIS12/fisChecks.ts

```diff
@@ -256,12 +257,12 @@ export const validateContext = (context: any, msgIdSet: any, pastCall: any, cure
       if (curentCall.startsWith('on_')) {
         logger.info(`Comparing Message Ids of /${pastCall} and /${curentCall}`)
         if (!_.isEqual(prevContext.message_id, context.message_id)) {
-          errorObj.message_id = `Message Id for /${pastCall} and /${curentCall} api should be same`
+          errorObj.message_id = `message_id for /${pastCall} and /${curentCall} api should be same`
```

Comment:
why is message_id of past and current call kept same? message_id should only be same for an API message pair.